src/commands/adivinhar.js

```diff
@@ -145,7 +147,7 @@ async function sendGameMessageAndResults(interaction) {
 
 		if (word === 'cancelar') {
 			await interaction.editReply('Você encerrou o jogo :(');
-			activeGames.splice(activeGames.indexOf(interaction.user.id), 1);
+			activeGames[interaction.user.id] = null;			
```

Comment:
```suggestion
			delete activeGames[interaction.user.id];
```
While this works, using `delete` is the correct way to do it, and the equivalent of `splice`.